ir/instr.cpp

```diff
@@ -593,6 +597,8 @@ StateValue UnaryOp::toSMT(State &s) const {
   case FNeg:
     fn = [](auto v) { return v.fneg(); };
     break;
+  default:
```

Comment:
I would prefer to keep Copy explicit and not have any default, since it makes it easier when you add a new opcode.